lib/uri_fetcher.rb

```diff
@@ -49,14 +54,18 @@ def handle_response(response)
         response.read_body
         recall
       else
-        response.read_body
-        msg = <<-MSG
-Can't follow the response from #{uri} anymore.
-        MSG
-        raise UnfollowableResponseError.new(msg, last_response: response)
+        try_error_handling_or_do(response) do
+          response.read_body
+          raise UnfollowableResponseError.new(last_response: response)
```

Comment:
Style/RaiseArgs: Provide an exception class and message as arguments to `raise`.